pmoves/services/agent-zero/mcp_server.py

```
@@ -211,6 +218,265 @@ def _stdout(msg: Dict[str, Any]):
+    return headers
+
+
+def _generate_error_id() -> str:
+    """Generate unique error ID for tracking."""
+    return str(uuid.uuid4())[:8]
+
+
+def _e2b_request(
+    method: str,
+    url: str,
+    json_body: Dict = None,
```

Ruff (0.14.10)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)